rest/taskana-rest-spring/src/main/java/pro/taskana/monitor/rest/assembler/ReportRepresentationModelAssembler.java

```diff
@@ -138,11 +138,11 @@ public ReportRepresentationModel toModel(
 
   @NonNull
   public ReportRepresentationModel toModel(
-      @NonNull TaskStatusReport report,
-      @NonNull List<String> domain,
-      @NonNull List<TaskState> state,
-      @NonNull List<String> workbasketIds,
-      @NonNull Integer priorityMinimum)
+      TaskStatusReport report,
```

Comment:
@mustaphazorgati what do you think about removing the @NotNull annotation? Currently, this toModel is called in a way that the parameters can be null. Sonarcloud sees this as bug. That's why the annotation doesn't seem to br correct to me. Is there a reason for this annotation? Is it an ok solution to just remove the annotation?